tests/basics/gc1.py

```diff
@@ -20,3 +20,10 @@
     # just test they execute and return an int
     assert type(gc.mem_free()) is int
     assert type(gc.mem_alloc()) is int
+
+if hasattr(gc, 'threshold'):
+    # uPy combines get/set_threshold function
```

Comment:
True, they are different from the implementation point of view. I will amend the comment.